src/main/java/logisticspipes/proxy/buildcraft/robots/boards/LogisticsRoutingBoardRobotNBT.java

```diff
@@ -41,7 +41,8 @@ public String getID() {
 
     @Override
     @SuppressWarnings({ "unchecked" })
-    public void addInformation(ItemStack stack, EntityPlayer player, List list, boolean advanced) {
+    public void addInformation(ItemStack stack, EntityPlayer player, @SuppressWarnings("rawtypes") List list,
```

Comment:
Why is this keeping the raw types?

Reply:
The parent class (`buildcraft.api.boards.RedstoneBoardNBT`) defines the method like this:
```java
public abstract void addInformation(ItemStack stack, EntityPlayer player, List<?> list, boolean advanced);
```
so the overriding method has to match that or use raw types. The former is not possible though because passing a String to `list.add()` wouldn't be then:
```
C:\MINECRAFT\dev\1.7.10\_GTNH\LogisticsPipes\src\main\java\logisticspipes\proxy\buildcraft\robots\boards\LogisticsRoutingBoardRobotNBT.java:45: error: incompatible types: String cannot be converted to CAP#1
        list.add(StringUtils.translate("robot.logisticsRouting"));
                                      ^
  where CAP#1 is a fresh type-variable:
    CAP#1 extends Object from capture of ?
```